scene/debugger/scene_debugger.cpp

```diff
@@ -378,7 +419,16 @@ void SceneDebugger::_set_object_property(ObjectID p_id, const String &p_property
 		prop_name = ss[ss.size() - 1];
 	}
 
-	obj->set(prop_name, p_value);
+	Variant value;
+	if (p_field.is_empty()) {
+		// Whole value.
+		value = p_value;
+	} else {
+		// Only one field.
+		value = fieldwise_assign(obj->get(prop_name), p_value, p_field);
```

Comment:
This seems to cause a compilation error in `template_debug`:

```
scene/debugger/scene_debugger.cpp:428:11: error: 'fieldwise_assign' was not declared in this scope
```

Reply:
What do you think would be the best course of action here? Wrapping it around `TOOLS_ENABLED` solves it, but that means per field value changing would be broken for remote debugging. So maybe making "math_fieldwise" be available for debugging templates would be the best alternative?

Reply:
Forgive the potentially dumb question, but can you actually remote debug an exported debug template? I can't say I've ever tried.

If so, then yeah, it doesn't strike me as a particularly heavy or otherwise dangerous function to expose, so I would probably opt for just exposing it for `DEBUG_ENABLED` as well. If you need it you need it.

Reply:
> Forgive the potentially dumb question, but can you actually remote debug an exported debug template? I can't say I've ever tried.

Yes, it's mostly for mobile debugging.

> If so, then yeah, it doesn't strike me as a particularly heavy or otherwise dangerous function to expose, so I would probably opt for just exposing it for DEBUG_ENABLED as well. If you need it you need it.

Gotcha.